share/nb.po

```diff
@@ -172,6 +177,13 @@ msgstr ""
 "--level må være en av CRITICAL, ERROR, WARNING, NOTICE, INFO, DEBUG, DEBUG2 "
 "eller DEBUG3.\n"
 
+msgid ""
+"Only one domain can be given for testing. Did you forget to prepend an "
+"option with '--<OPTION>'?\n"
+msgstr ""
+"Bare ett domene kan gis for testing. Glemte du å sette '--<OPTION>' foran en "
+"opsjon?"
```

Comment:
missing newline at the end

Reply:
Good catch! The trailing new lines are crap.